pkgs/by-name/fl/flatpak/package.nix

```diff
@@ -74,10 +74,12 @@ stdenv.mkDerivation (finalAttrs: {
       "out"
       "dev"
     ]
-    ++ lib.optionals (withDocbookDocs || withGtkDoc) [
-      "devdoc"
+    ++ lib.optionals withDocbookDocs [
```

Comment:
One small thing: I don't like how the first commit combines those two outputs under one option and the next one moves devdoc under another condition. Would probably be better to do this in one.

Reply:
Would you be happy if I just squashed those two commits together?